### DISCUSSION TITLE
Manage Plugins menu not workingg

### DISCUSSION THREAD

**Discussion by gnfb1234** [YOURLS/YOURLS#3563] [Plugins]
I have now installed Yourls every which way I can itty.be got the https working was able to manually upload a plugin and have it show up correctly tried changeing the php to 8.2 BUT the Manage Plugins drop down menu will not work / drop down so I can activate plugins
any ideas about to give up and go to a horrible commercial version

**Comment by ozh** [comment]
There is no "manage plugin drop down".

You don't accurately describe the problem, so nobody will be able to help. See https://yourls.org/docs/category/troubleshooting
